app/assets/javascripts/formValidate.js

```diff
@@ -0,0 +1,175 @@
+function validateSummonerField() {
+  var id = 'summoner_name'
+  $(".email").hide();
+  return fieldValidation(id);
+};
+function validateConfirmSummonerField() {
+  var id = 'confirm_summoner_name'
+  $(".email").hide();
+  return fieldValidation(id);
+};
+function validateEmailField() {
+  var id = 'summoner_email'
+  return fieldValidation(id);
+};
+function validateConfirmEmailField() {
+  var id = 'confirm_summoner_email'
+  return fieldValidation(id);
+};
+function fieldValidation(id) {
+  clearFeedback()
+  var fieldText = $("#"+id).val();
+  if (fieldText === "" || fieldText.length <3 || fieldText.length > 24){
+    $("#"+id+"_feedback").text("Required field, character length between 3 and 24.");
+    $("#"+id+"_feedback").css("color","red");
```

Comment:
Don't add your css through JS unless it's absolutely necessary.  You can add the color to your `.feedback` **class** in the stylesheet.  That way you also only need to specify it once instead of declaring it for each div with a different id.